src/lib/RpcApi.ts

```diff
@@ -28,8 +28,13 @@ import CookieJar from '@/lib/CookieJar';
 import { Raven } from 'vue-raven'; // Sentry.io SDK
 import { ERROR_CANCELED } from './Constants';
 import Config from 'config';
+import { setHistoryStorage, getHistoryStorage } from '@/lib/Helpers';
 
 export default class RpcApi {
+    private static get HISTORY_KEY_RPC_STATE() {
+        return 'rpc-state';
```

Comment:
Careful with naming here, since the RPC lib internally also has a class called `State`, which is sometimes imported as `RpcState`, but here in the `RpcApi` class the State means something more and stores all of the Hub state. So I suggest calling it `hub-state`.